README.md

```diff
@@ -63,8 +68,12 @@
 
 For a live demo to our project, follow the link below
 
+
+- [Live Demo Link](https:https://github.com/maherramadan78/leaderboard)
```

Comment:
- [ ] This link is broken. kindly consider removing the link as broken links make our readme look more unprofessional.